### ISSUE TITLE
Captcha does not appear on the new user signup page 

### ISSUE THREAD

**Issue by benrfairless** [openaustralia/righttoknow#775]
If you go to https://www.righttoknow.org.au/profile/sign_in?r=%2F you don't get prompted for a captcha if you create a new account.

~~This works in test, so there must be a theme issue in production.~~

Captchas appear everywhere else in production just fine.

**Comment by benrfairless** [comment]
I've traced this to here: https://github.com/mysociety/alaveteli/blob/74834d1b2deab1a99ae1c69a42cc6a2ff5d31990/app/views/user/_signup.html.erb#L44

I suggest we remove this line as we are still getting spammy users being created:
`<% if @request_from_foreign_country %>`

**Comment by mlandauer** [comment]
@benrfairless do we know for sure that those spammy users are being created from within Australia?

**Comment by benrfairless** [comment]
@mlandauer I don't think we record the IP addresses of sign ups. If we do, I'll provide some recent example accounts and we can see if they are Australian or not.

**Comment by mlandauer** [comment]
@benrfairless can you send me some links to recent (since we've been on the latest version of Alaveteli) spam accounts in the admin panel and then I can go and investigate to see if I can figure out where they're coming from?

**Comment by benrfairless** [comment]
Have sent you a few examples in slack to review 😇

**Comment by mlandauer** [comment]
Just for completeness I accessed the production site through a VPN and as expected it showed the recaptcha on the user sign up. I also checked that the form doesn't work when the recaptcha isn't completed.

So, after much trawling I figured out that the spam accounts that @benrfairless sent me for review were both created by requests originating outside Australia and they both successfully completed the recaptcha. There's not much we can do if spammers are paying people (or clever computer programs) to fill out recaptchas.

So, I'm closing this issue
